etl/steps/data/meadow/hyde/2024-01-03/general_files.py

```diff
@@ -0,0 +1,57 @@
+"""Load a snapshot and create a meadow dataset."""
+import os
+from pathlib import Path
+
+import owid.catalog.processing as pr
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("general_files.zip")
+
+    # Load data from snapshot.
+    with snap.extract_to_tempdir() as tmpdir:
+        # Get all files in the directory
+        files = os.listdir(tmpdir)
+        # Sanity check
+        fname_expected = "HYDE_country_codes.xlsx"
+        assert fname_expected in files, f"The directory should contain a file named {fname_expected} (empty string)!"
+        # Read file
+        tb = pr.read_excel(
+            Path(tmpdir) / fname_expected,
+            metadata=snap.to_table_metadata(),
+            origin=snap.metadata.origin,
+        )
+
+    #
+    # Process data.
+    #
+    # Keep good columns
+    tb = tb[["ISO-CODE", "Country"]]
+
+    # Filter spurious entries
+    mask = (tb["ISO-CODE"] == 826) & (tb["Country"] != "United Kingdom  ")
```

Comment:
I'd include an assertion to check that the spurious rows exist, so that we remove this part of the code in the future, if they are cleaned. Also, shouldn't "United Kingdom  " be mapped to "United Kingdom"? (I see a similar mapping in the country file for "Afghanistan ").